Resources/public/js/views/services/ez-languageselectionboxviewservice.js

```diff
@@ -30,24 +30,16 @@ YUI.add('ez-languageselectionboxviewservice', function (Y) {
         ATTRS: {
             /**
              * List of available translations. List contains language objects.
+             * It is hash indexed by languageCode.
              *
              * @attribute availableTranslations
              * @type {Object}
+             * @readOnly
              */
             availableTranslations: {
-                value: {
-                    'eng-GB': {id: 2, languageCode: 'eng-GB', name: 'English (United Kingdom)', enabled: true},
-                    'nno-NO': {id: 4, languageCode: 'nno-NO', name: 'Norwegian (Nynorsk)', enabled: true},
-                    'chi-CN': {id: 8, languageCode: 'chi-CN', name: 'Simplified Chinese', enabled: true},
-                    'cze-CZ': {id: 16, languageCode: 'cze-CZ', name: 'Czech', enabled: true},
-                    'eng-US': {id: 32, languageCode: 'eng-US', name: 'English (American)', enabled: true},
-                    'esl-ES': {id: 64, languageCode: 'esl-ES', name: 'Spanish (Spain)', enabled: true},
-                    'fre-FR': {id: 128, languageCode: 'fre-FR', name: 'French (France)', enabled: true},
-                    'ita-IT': {id: 256, languageCode: 'ita-IT', name: 'Italian', enabled: true},
-                    'jpn-JP': {id: 512, languageCode: 'jpn-JP', name: 'Japanese', enabled: true},
-                    'swe-SE': {id: 1024, languageCode: 'swe-SE', name: 'Swedish', enabled: true},
-                    'pol-PL': {id: 2048, languageCode: 'pol-PL', name: 'Polish', enabled: true},
-                    'ger-DE': {id: 4096, languageCode: 'ger-DE', name: 'German', enabled: true},
+                readOnly: true,
+                valueFn: function () {
+                    return this.get('app').get('systemLanguageList');
```

Comment:
actually you should even completely remove this attribute and change the `_getViewParameters` implementation to directly use `this.get('app').get('systemLanguageList')`